src/account/accountSelectors.js

```diff
@@ -1,5 +1,5 @@
 export const getActiveAccount = (state) =>
-  state.account[0];
```

Comment:
I realized the problem now. If `account` isn't a key in `state` then this throws an exception whereas the updated line returns `undefined`.

This occurs when rehydrating from an empty state because I call `!!getAuth(action.payload).apiKey` and `action.payload` is empty in that case